src/main/java/codoc/logic/parser/AddCommandParser.java

```diff
@@ -40,22 +45,26 @@ public AddCommand parse(String args) throws ParseException {
                         PREFIX_GITHUB, PREFIX_EMAIL,
                         PREFIX_LINKEDIN,
                         PREFIX_SKILL,
-                        PREFIX_MOD
+                        PREFIX_MOD,
+                        PREFIX_COURSE,
+                        PREFIX_YEAR
                 );
 
-        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_EMAIL)
+        if (!arePrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_EMAIL, PREFIX_YEAR, PREFIX_COURSE)
```

Comment:
Year and course has been added as a compulsory parameter for AddPerson.
Then, compulsory parameters are name, email, year, course.